apps/prairielearn/elements/pl-multiple-choice/pl-multiple-choice.mustache

```diff
@@ -50,6 +50,51 @@
 {{#inline}}
 </span>
 {{/inline}}
+{{/radio}}
+{{^radio}}
+<!-- Weirdly enough putting this in a div causes it to display without the inline -->
+<span class="py-1 d-inline-flex align-items-center">
+    <script>
+        $(() => {
+            $('#pl-multiple-choice-select-{{uuid}}').selectpicker({})
+                .on('rendered.bs.select loaded.bs.select shown.bs.select', (e) => {
+                    // If any of the options has math, render the math as needed
+                    // Events: loaded (when the page opens), rendered (when the value changes),
+                    // shown (when the dropdown extends)
+                    MathJax.typesetPromise();
+                });
+        });
+    </script>
+    <select name="{{name}}" data-style="btn-outline-secondary"
```

Comment:
One more comment, I actually like this style less than the default shown on https://developer.snapappointments.com/bootstrap-select/examples/:

<img width="245" alt="Screenshot 2024-02-07 at 11 51 14" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/e18255c6-3835-43ed-abea-2c5a824c53f4">

Is there a reason you went with this style?

Reply:
I thought the input box looked weird without a border around it. Maybe personal preference but for something we want students to answer, I think the box should stand out a little from other text (but not a strong feeling).

Reply:
Yeah, I agree that a border looks nice. I got good results with the following:

```
data-style="btn-light border border-secondary"
```

Let me know what you think of that?

Reply:
Tried it out and I think it looks good! (see screenshot) Keeps the border while making the displayed HTML not look faded.

![image](https://github.com/PrairieLearn/PrairieLearn/assets/1345068/4e8a033f-a0fa-48c4-8d1a-c28b12616817)

Changes committed here: https://github.com/PrairieLearn/PrairieLearn/pull/8903/commits/67b78635217ede9f189d2234748f62cb03cbaba4